src/cuttlefish_conf.erl

```diff
@@ -122,6 +123,11 @@ generate_element(MappingRecord) ->
             lager:warning("{level, ~p} has been deprecated. Use 'hidden' or '{hidden, true}'", [Level])
     end,
 
+    Default = case ConfFileDefault of
```

Comment:
So I like the general idea, but in this case changing the behavior of `cuttlefish_mapping:config_file_default/1` would break the pattern of having simple accessor functions in `cuttlefish_mapping` that match the name of the attribute. How about if I add a helper function in `cuttlefish_conf` to bridge the gap instead?